File: app/src/main/java/com/callmangement/EHR/models/ModelSurveyFormListing.java
package com.callmangement.EHR.models;

import com.google.gson.annotations.SerializedName;

import java.util.ArrayList;
import java.util.List;

public class ModelSurveyFormListing {
    @SerializedName("status")
    private String status;
    @SerializedName("message")
    private String message;
    @SerializedName("Data")
    private List<SurveyFormDetailsInfo> list = new ArrayList<>();

    public String getStatus() {
        return status;
    }

    public void setStatus(String status) {
        this.status = status;
    }

    public String getMessage() {
        return message;
    }

    public void setMessage(String message) {
        this.message = message;
    }

    public List<SurveyFormDetailsInfo> getList() {
        return list;
    }

    public void setList(List<SurveyFormDetailsInfo> list) {
        this.list = list;
    }
}

File: app/src/main/java/com/callmangement/ehr/models/ModelSurveyFormListing.kt
package com.callmangement.ehr.models

import com.google.gson.annotations.SerializedName

class ModelSurveyFormListing {
    @JvmField
    @SerializedName("status")
    var status: String? = null

    @JvmField
    @SerializedName("message")
    var message: String? = null

    @JvmField
    @SerializedName("Data")
    var list: List<SurveyFormDetailsInfo> = ArrayList()
}
